src/aind_dynamic_foraging_models/logistic_regression/model.py

```diff
@@ -285,7 +317,11 @@ def _bootstrap(func, X, Y, n_iters=1000, n_samplesize=None, **kwargs):
     bootstrap_X = [X[index, :] for index in indices]
 
     # Apply func to each bootstrap sample
-    return np.array([func(X, Y, **kwargs) for X, Y in zip(bootstrap_X, bootstrap_Y)])
+    results = []
+    for X, Y in zip(bootstrap_X, bootstrap_Y):
+        if len(np.unique(Y)) > 1:
+            results.append(func(X, Y, **kwargs))
```

Comment:
Another possibility is to run a while loop until n_iters number of _valid_ bootstraps are drawn, which keeps the n_iters the same. Not sure which one is better.

Reply:
The worst case scenario is that we only have one left choice out of window_size trials. That would lead to a high number of bootstraps getting rejected, so the confidence intervals would be computed from a small number of bootstraps. But, the confidence intervals would be hard to interpret from a scenario with only one left choice anyways. 

I'm going to merge it in, but it is an edge case we should keep in mind.

Reply:
Made an issue for the record: https://github.com/AllenNeuralDynamics/aind-dynamic-foraging-models/issues/49